react-app/src/components/SearchForm/index.tsx

```diff
@@ -66,7 +66,7 @@ export const SearchForm: FC<{
         )}
         {!!searchText && (
           <XIcon
-            className="absolute cursor-pointer top-0 bottom-0 w-6 h-6 my-auto right-0 lg:left-[28rem] xl:left-[55%]"
+            className="absolute cursor-pointer top-0 bottom-0 w-6 h-6 my-auto right-0 lg:left-[28rem]"
             data-testid="close-icon"
             onClick={() => {
               setSearchText("");
```

Comment:
Out of scope, but this should ideally be a button

Reply:
#todo

Reply:
put it in the accessibility fix list 😄